src/CodeReviews.coffee

```diff
@@ -137,14 +137,20 @@ class CodeReviews
   # @param String origin_room string of origin room
   # @param String channel_to_notify string of the user/room to notify
   notify_channel: (cr, origin_room, channel_to_notify) ->
-    attachments = []
-    attachments.push
-      fallback: "#{cr.url} could use your :eyes: Remember to claim it in ##{origin_room}"
-      text: "*<#{cr.url}|#{cr.slug}>* could use your :eyes: Remember to claim it" +
-      " in <https://slack.com/app_redirect?channel=#{origin_room}|##{origin_room}>"
-      mrkdwn_in: ["text"]
-      color: "#575757"
-    sendFancyMessage @robot, channel_to_notify, attachments
+    slackTeamId.then (teamId) ->
```

Comment:
I think this will need parameters? 

```
module.exports = (msg, next) -> new Promise (resolve, reject) ->
```

implies you'll need to supply at least msg?

Reply:
![Screen Shot 2022-09-27 at 10 39 42 AM](https://user-images.githubusercontent.com/7146063/192585486-c11f17c3-620b-4868-9839-14df35b81cff.png)